sys/include/xtimer.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
  * @note This is a struct in order to make the xtimer API type strict
  */
 typedef struct {
-    uint64_t ticks64;
+    uint64_t ticks64;       /**< Captain Obvious to the rescue! */
```

Comment:
while funny, I don't think this text should be like this. "Tick count" or something will do.